packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js

```diff
@@ -35,20 +35,17 @@ export function NativeValue({ data, edit, onSelect, onChange, jsonpath }) {
 		return <input type={inputType} value={data} onChange={e => onChange(e, { jsonpath })} />;
 	}
 
-	const lineValueClasses = classNames(
-		theme.native,
-		theme[type],
-		theme['line-value'],
-	);
+	const lineValueClasses = classNames(theme.native, theme[type], theme['line-value']);
 
 	return (
-		<button
+		<span
 			className={lineValueClasses}
-			type="button"
+			role="button"
+			tabIndex="0"
```

Comment:
Could you add `onKeyUp` event management too? `onKeyUp={e => stopAndSelectWithEnterOrSpace(e, { onSelect, jsonpath })}`